integration-tests/tests/run-workflow-on-snapshot.js

```diff
@@ -53,15 +53,13 @@ const testRunWorkflowOnSnapshotFn = _.flow(
 
   await click(page, clickable({ textContains: 'Inputs' }))
   await fillInReplace(page, input({ labelContains: 'echo_to_file input1 attribute' }), `this.${snapshotColumnName}`)
-  await delay(1000) // Without this delay, the input field sometimes reverts back to its default value
-  await click(page, clickable({ text: 'Save' }))
 
   await click(page, clickable({ textContains: 'Outputs' }))
   await fillInReplace(page, input({ labelContains: 'echo_to_file out attribute' }), 'workspace.result')
-  await delay(1000) // Without this delay, the input field sometimes reverts back to its default value
-  await click(page, clickable({ text: 'Save' }))
 
-  await delay(1000) // The Run Analysis button (launchWorkflowAndWaitForSuccess) requires time to become enabled after hitting the save button
+  await click(page, clickable({ text: 'Save' }))
```

Comment:
When running 100s of times locally, it was clear that the test only was failing on the second Save attempt.

Save will persist both the inputs and outputs, so there was no reason to press Save twice. There may be a race condition between the page update after the Save response is received and editing the output field. I doubt that a user would ever encounter it though because they would not be able to interact with the elements that quickly. I was not able to reproduce it in my manual tests.

Reply:
Feels great to see those delays removed  🤩

Reply:
Good find! 
I wonder if something changed in the product leading to the dramatical increase of the failure rate, and if there are implications for other tests. But that's obvs out of scope for this PR.